app/Repositories/TopicRepository.php

```diff
@@ -98,7 +105,11 @@ public function sortByLetter($topics)
     public function setSelected($topics, $topic)
     {
         return collect($topics)->map(function ($item) use ($topic) {
-            $item['selected'] = $item['slug'] === $topic ? true : false;
+            if (!empty($topic)) {
+                $item['selected'] = $item['slug'] === $topic ? true : false;
```

Comment:
these can be returned instead of doing a ternary:

$item['selected'] = $item['slug'] === $topic;

$item['selected'] = $item['slug'] === '';

Reply:
Awesome, I'll work on this. Thanks!